app/models/status.rb

```diff
@@ -393,6 +415,9 @@ def unlink_from_conversations!
   def update_status_stat!(attrs)
     return if marked_for_destruction? || destroyed?
 
+    attrs[:untrusted_favourites_count] = attrs[:favourites_count] if attrs.key?(:favourites_count) && !untrusted_favourites_count.nil?
+    attrs[:untrusted_reblogs_count] = value if key == :favourites_count && !untrusted_favourites_count.nil?
```

Comment:
I think `value` is a typo here.

Reply:
oops leftovers from a prior attempt, reverting